bedrock/mozorg/templatetags/misc.py

```diff
@@ -226,6 +226,36 @@ def high_res_img(ctx, url, optional_attributes=None):
     return jinja2.Markup(markup)
 
 
+@library.global_function
+@jinja2.contextfunction
+def lazy_img(ctx, image_url, placeholder_url, include_highres_image=False, optional_attributes=None):
+    placeholder = static(path.join('img', placeholder_url))
+    image = static(path.join('img', image_url))
+
+    if include_highres_image:
+        image_high_res = static(path.join('img', convert_to_high_res(image_url)))
+        srcset = 'data-srcset="{image_high_res} 2x"'.format(image_high_res=image_high_res)
+    else:
+        srcset = ''
+
+    if optional_attributes:
+        class_name = optional_attributes.pop('class', 'lazy-image')
+        attrs = ' '.join('%s="%s"' % (attr, val)
+                         for attr, val in optional_attributes.items())
+    else:
+        class_name = 'lazy-image'
+        attrs = ''
+
```

Comment:
Maybe add a check here for an `alt` optional attribute, and, if not present, default it to an empty string? Could save a fair amount of repetition in the template calls.